Storage partitioning updates (#6305)
GoogleChrome/developer.chrome.com: site/en/docs/privacy-sandbox/storage-partitioning/index.md

layout: "layouts/blog-post.njk"
title: "Storage Partitioning"
subhead: >
  To prevent certain types of side-channel cross-site tracking, Chrome is partitioning storage and communications APIs in third-party contexts.
description: >
  To prevent certain types of side-channel cross-site tracking, Chrome is partitioning storage and communications APIs in third-party contexts.
authors:
 - kevinkiklee
 - mihajlija
date: 2022-08-24
updated: 2023-05-16
tags:
 - privacy
---
 
## Implementation status

- Full implementation is available for testing in Chrome Beta 113 and later.
- The initial implementation has been [available behind a flag since Chrome 105](/blog/storage-partitioning-dev-trial/).
- [Storage Partitioning proposal](https://github.com/privacycg/storage-partitioning) is open for discussion.
- [Chrome Platform Status](https://chromestatus.com/feature/5723617717387264)

The feature is rolling out to a small percentage of users starting in Chrome 113. 

> Depending on the stability and compatibility, it will be made fully available in
> Chrome Stable by the end of June 2023. Testing third-party storage partitioning now

Author: ```suggestion
Depending on the stability and compatibility, it will be made fully available in
Chrome Stable mid-year 2023. Testing third-party storage partitioning now
```
